client/flutter/lib/main.dart

```diff
@@ -30,7 +33,25 @@ void main() async {
   final bool onboardingComplete =
       await UserPreferences().getOnboardingCompleted();
 
-  runApp(MyApp(showOnboarding: !onboardingComplete));
+  if (onboardingComplete) {
+  // Set `enableInDevMode` to true to see reports while in debug mode
+    // This is only to be used for confirming that reports are being
+    // submitted as expected. It is not intended to be used for everyday
+    // development.
+    Crashlytics.instance.enableInDevMode = false;
+
+    // Pass all uncaught errors from the framework to Crashlytics.
+    FlutterError.onError = Crashlytics.instance.recordFlutterError;
```

Comment:
Still need to https://firebase.google.com/docs/crashlytics/customize-crash-reports?authuser=0&platform=ios#enable_opt-in_reporting